vue/vite.config.ts

```diff
@@ -71,6 +71,10 @@ export default ({ mode}) => {
             target: `http://${devHost}:8000`,
             xfwd: true,
           },
+          "/static": {
+            target: `http://${devHost}:8000`,
+            xfwd: true,
+          },
```

Comment:
It was leftover from testing on this PR: https://github.com/ResonantGeoData/RD-WATCH/pull/460
When you aren't logged in it would transfer you to the login page and this would allow redirection of the static assets.  I'll remove it and wait until that PR is integrated.